docs/metrics-storage-management/reference-streaming.mdx

```diff
@@ -367,7 +367,7 @@ Note: The `stream-compression` status can be `"enabled" | "disabled" | "N/A"`.
 A compressed data packet is determined and decompressed on the fly.
 
 #### Limitations
-This limitation will be withdrawn asap and is work-in-progress.
+This limitation will be withdrawn ASAP and is a work-in-progress.
```

Comment:
I think that is a famous Brazilian ASAP (cc @thiagoftsm), I think we better remove the line 😄

Reply:
We can just say "in a future update" or "in a future release"